package.json

```diff
@@ -4,23 +4,23 @@
   "description": "Utility library that extracts the CLDR data that's needed by the FormatJS libraries.",
   "main": "index.js",
   "scripts": {
-    "test": "istanbul cover -- _mocha test/unit/ --reporter spec"
+    "test": "nyc mocha test/unit/ --reporter spec"
   },
   "author": "Eric Ferraiuolo <edf@ericf.me>",
   "license": "BSD-3-Clause",
   "dependencies": {
-    "cldr-core": "^34.0.0",
-    "cldr-dates-full": "^34.0.0",
-    "cldr-numbers-full": "^34.0.0",
-    "glob": "^5.0.1",
-    "make-plural": "^2.1.3",
+    "cldr-core": "34",
+    "cldr-dates-full": "34",
+    "cldr-numbers-full": "34",
+    "glob": "7",
+    "make-plural": "4",
```

Comment:
Why did you choose not to use semver? Personally, I feel uncomfortable.

Reply:
`34` is semver, it's the same as `^34.0.0`. I personally don't mind either way, just less chars. I can change to that if you want 😄